file_repository/file_connexion.py

```diff
@@ -197,3 +197,19 @@ def move(self, oldfilepath, newfilepath, filename, new_filename=None):
         elif self.is_('filestore'):
             os.rename(os.path.join(oldfilepath, filename),
                       os.path.join(newfilepath, new_filename))
+
+    @open_and_close_connection
+    def delete(self, filepath, filename):
+        if not filepath: filepath = ''
+        if filename is False:
+            raise NoFileNameExcept("Filename is not defined")
```

Comment:
Actually, I just copy the raise from another method, but I think the best message may have been the first one!
If filename is False, no need to show the filepath, because it is not related. Indeed we can't delete a file without a file name.